test/extended/dr/machine_recover.go

```diff
@@ -155,7 +175,102 @@ var _ = g.Describe("[Feature:DisasterRecovery][Disruptive]", func() {
 					return true, nil
 				})
 				o.Expect(err).NotTo(o.HaveOccurred())
+
+				framework.Logf("Verify recreated master has healthy etcd endpoint")
+				healthyEtcds = getHealthyEtcds(certsDir, masters)
+				o.Expect(len(healthyEtcds)).To(o.BeNumerically("==", len(masters)))
+
 			})
 	},
 	)
 })
+
+func getHealthyEtcds(certsDir string, masters []*corev1.Node) []*etcdserverpb.Member {
```

Comment:
IMO these helper functions should be in common.go, will add a TODO

Reply:
Note that we don't want big bundles of helpers - there's not a ton of overlap between this method and the other methods, so placing helpers specific to a specific location is ok for now and refactoring later is fine.